web/client/plugins/querypanel/SpatialFilter.jsx

```diff
@@ -0,0 +1,429 @@
+/**
```

Comment:
This is a copy connected of SpatialFilter in `/components/data/query/SpatialFilter.jsx` . Please reuse the component.